src/share/examples/poudriere/hiawatha.conf.samples

```diff
@@ -0,0 +1,12 @@
+Please copy this to your hiwatha.conf
```

Comment:
Dumb question but does Hiawatha allow comments anywhere like this? It's also spelled wrong.